### DISCUSSION TITLE
enforce 'u' flag in all regexp?

### DISCUSSION THREAD

**Discussion by kaizhu256** [jslint-org/jslint#353] [Ideas]
@douglascrockford what do you think about enforcing 'u' flag in all regexps?

have a working patch @ https://github.com/kaizhu256/jslint/compare/dc150a3f978b04672beab331d75831f3552b6489..5998bd7ab5539c15e025509086f1f4a30a2a9398#diff-af28e552c2ace6249b1da210db6c7ded420c302be144da1bd86337e6bbdee19bR2158

and demo @ https://kaizhu256.github.io/jslint/branch-alpha/index.html

**Comment by douglascrockford** [comment]
I don't see the value of it. It allows some new regexp forms, but if you are not using those forms, it is just unnecessary specification. It is a breaking change with no obvious benefit.

**Reply by kaizhu256** [reply]
ok thx.
